src/transformers/models/albert/tokenization_albert.py

```diff
@@ -102,6 +102,20 @@ class AlbertTokenizer(PreTrainedTokenizer):
         mask_token (:obj:`str`, `optional`, defaults to :obj:`"[MASK]"`):
             The token used for masking values. This is the token used when training this model with masked language
             modeling. This is the token which the model will try to predict.
+        sp_model_kwargs (:obj:`dict`, `optional`, defaults to :obj:`None`):
```

Comment:
```suggestion
        sp_model_kwargs (:obj:`dict`, `optional`):
```
We don't specify the default when it's None.

I'm putting it once here, but it applies to every other docstring as well.

Reply:
I changed this and created a new Issue as a reminder to apply the same change on other locations: https://github.com/huggingface/transformers/issues/11687